apps/frontend/src/views/Results.vue

```diff
@@ -297,6 +330,58 @@ export default class Results extends mixins(RouteMixin, ServerMixin) {
   /** Determines if we should make the search bar collapse-able */
   show_search_mobile: boolean = false;
 
+  /** If we are currently showing the search help modal */
+  showSearchHelp = false;
+
+  @Watch('searchTerm')
+  onUpdateSearch(_newValue: string) {
+    if (this.typingTimer) {
+      clearTimeout(this.typingTimer);
+    }
+    this.typingTimer = setTimeout(this.onDoneTyping, 1000);
+  }
+
+  @Watch('isTyping')
+  onDoneTyping() {
+    this.clear()
+    const searchResult: SearchQuery[] = parse(this.searchTerm)[0];
+    searchResult.forEach((result) => {
```

Comment:
It may be easier to just have computed properties for these, especially if all the data is going to be contained in the search bar anyway if this is updated to automatically update the search bar when users click on different filters on the page.

Reply:
I believe it require be a lot of duplication to have similar parsers for each filter type, and now that search has its own store I don't think it will affect as much.